debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -83,6 +86,22 @@
         .withImportance(Importance.LOW)
         .withDescription("For default oracle 12+, use default pos_version value v2, for oracle 11, use pos_version value v1.");
 
+    public static final Field CONNECTOR_ADAPTER = Field.create(DATABASE_CONFIG_PREFIX + "connection.adapter")
+            .withDisplayName("Connector adapter")
+            .withEnum(ConnectorAdapter.class, ConnectorAdapter.XSTREAM)
+            .withWidth(Width.MEDIUM)
+            .withImportance(Importance.HIGH)
+            .withDescription("There are two adapters: XStream and LogMiner.");
+
+    // todo remove this after merging changes on sqlserver
+    public static final Field ROWS_FETCH_SIZE = Field.create("rows.fetch.size")
```

Comment:
A common open has been introduced for this recently in the shared base class: `snapshot.fetch.size`.